### DISCUSSION TITLE
Where it's better to place wine commands?

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [tldr-pages/tldr#7411] [Q&A] [answered]
Hello! Where it's better to place wine commands? To linux or windows folder?

**Comment by navarroaxel** [comment, marked as answer]
:thinking: You run `wine` commands in Linux terminal, so it's `linux/`

If the command is available in more than 1 platform, e.g. Linux & macOS, it should be placed on `common/`
